counsel.el

```diff
@@ -5138,6 +5141,37 @@ The optional `BUILD-ROOT' is useful for other helpers that have found
      local-files)
     compiles))
 
+(defun counsel--find-build-subdir (root)
+  "Return builds sub-directory of `ROOT', if one exists."
```

Comment:
I don't think we should enclose argument names in documentation string into quotes `` `...' ``.  That is uppercase is enough and that applies to all of your documentation strings.

Reply:
Indeed, arguments are never quoted in Elisp docstrings; see [`(elisp) Documentation Tips`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html).

Reply:
I think all arguments are now JUST CAPS.